<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckPayment</class>
 <widget class="QWidget" name="CheckPayment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CheckPayment</string>
  </property>
  <widget class="QTableView" name="productsListTbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>421</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>510</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="paymentBtn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>510</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Payment</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>460</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="receivedLayout">
    <item>
     <widget class="QLabel" name="moneyLbl">
      <property name="text">
       <string>Money received:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="moneyNumLbl">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>420</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="priceLayout">
    <item>
     <widget class="QLabel" name="priceLbl">
      <property name="text">
       <string>Total price:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="priceNumLbl">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="emulatedAddMoneyBtn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>450</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Emulated add 10$</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
